internal/cli/install/install.go

```diff
@@ -97,6 +97,10 @@ func Install(ctx context.Context, w io.Writer, k8sCfg *kubex.ConfigWithMeta, opt
 	if err != nil {
 		return err
 	}
+	if rel == nil {
+		//User answered no on prompt
```

Comment:
Could you please elaborate here? On which prompt? Please update the comment to clarify which case it covers. Thanks!